boards/arm/nucleo_f429zi/nucleo_f429zi.dts

```diff
@@ -7,6 +7,9 @@
 /dts-v1/;
 #include <st/stm32f429.dtsi>
 
+/* Temp: define arduino_i2c */
```

Comment:
There should exist a github issue for lack of the DTC feature support in the SDK and this comment should be stating that this is a workaround around that Github issue.